src/FormBuilder/resources/js/form-builder/src/styles/_components.scss

```diff
@@ -331,7 +390,29 @@ This creates a consistent separator between the tabs and the rest of the sidebar
 }
 
 .givewp-classic-editor {
```

Comment:
do we still need the classic editor styles now that we have the one form the form-builder-library?

Reply:
Resolved: a06389abd4c864b54cdec045e64be17db14e20ea